yt_dlp/extractor/rozhlas.py

```diff
@@ -45,3 +47,54 @@ def _real_extract(self, url):
             'duration': duration,
             'vcodec': 'none',
         }
+
+
+class RozhlasVltavaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:(?:vltava|wave|radiozurnal|dvojka|plus|sport|d-dur|jazz|junior|pohoda)\.rozhlas|english\.radio)\.cz/[a-z|-]*-(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://wave.rozhlas.cz/papej-masicko-porcujeme-a-bilancujeme-filmy-a-serialy-ktere-letos-zabily-8891337',
+        'md5': 'ba2fdbc1242fc16771c7695d271ec355',
+        'info_dict': {
+            'id': '8891337',
+            'ext': 'mp3',
+            'title': 'md5:1c6d29fb9564e1f17fc1bb83ae7da0bc',
+            'description': 'md5:bbab1431c3472304b4863faea414cd2b',
+            'duration': 1574
+        }
+    }]
```

Comment:
Definition is wrong. This happens to pass the test due to a quirk (bug?) in the test runner. Add `playlist` key to the definition. E.g. https://github.com/yt-dlp/yt-dlp/blob/ce03e9b90acef7a8b59b1193e32cbd9898c93ad0/yt_dlp/extractor/afreecatv.py#L51-L79

Reply:
This should also fix what you are saying about id

Reply:
done

The id problem was a bug on my side. 

I added a test for series download.